src/components/EditorMarkdownIt.vue

```diff
@@ -61,6 +61,7 @@ export default {
 				}
 
 				token.attrs[aIndex][1] = path
+				path = path.toLowerCase()
```

Comment:
This breaks the download if the file name has upper case letters. Please use a new variable (e.g. `lowecasePath`) for this.